helix-core/src/test/java/org/apache/helix/controller/rebalancer/waged/model/TestAssignableReplica.java

```diff
@@ -102,6 +103,24 @@ public void testConstructReplicaWithResourceConfig() throws IOException {
     Assert.assertEquals(replica.getResourceMaxPartitionsPerInstance(), maxPartition);
   }
 
+  @Test
```

Comment:
Nit: I think adding some JavaDoc here would be nice: 

E.g.) Tests that if default partition weight map is configured in ClusterConfig and NOT in ResourceConfig, AssignableReplica actually will get the default weight from ClusterConfig even though it's not set in ResourceConfig.